katello-pull-transport-migrate

```diff
@@ -56,14 +52,13 @@ log-level = "error"
 EOF
 
 else
-    echo "$SYSCONFDIR/yggdrasil/config.toml not found! Did `yum install yggdrasil` succeed?"
+    echo "$SYSCONFDIR/yggdrasil/config.toml not found! Did 'yum install yggdrasil' succeed?"
     exit 1
 fi
 
 # start the yggdrasild service
 echo "Starting yggdrasild..."
-systemctl enable yggdrasild
-systemctl start yggdrasild
+systemctl enable --now yggdrasild
```

Comment:
This made me wonder but it doesn't implement the systemd ready protocol so I opened https://github.com/RedHatInsights/yggdrasil/issues/56.

Together with the fail early code this may mean the script ends its execution here, depending on how the `yggdrasild` service behaves.